Tests/SwiftParserTest/translated/SwitchTests.swift

```diff
@@ -1402,11 +1179,7 @@ final class SwitchTests: XCTestCase {
       }
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 2: switch must be exhaustive
-        // TODO: Old parser expected note on line 2: do you want to add a default clause?
-        // TODO: Old parser expected error on line 5: unknown attribute 'unknown'
         DiagnosticSpec(message: "expected declaration after attribute in switch case"),
```

Comment:
Seems like we should parse this as the start of a `default` clause rather than a decl?

Reply:
I improved the diagnostic.